### PR TITLE
UF-7867: Corrections to the API

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: src/main/java/se/sundsvall/comfactfacade/api/model/Signatory.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(setterPrefix = "with")
+@Schema(description = "A Signatory related to the signing process.")
+public class Signatory extends Party {
+
+
+	@NotEmpty
+	@ArraySchema(schema = @Schema(implementation = Identification.class, description = "The means of identification to use to identify the signatories."))
```
Comment (mha109) [inline src/main/java/se/sundsvall/comfactfacade/api/model/Signatory.java:26]: I may be wrong, but is the description necessary here, won't it be taken from the `@Schema` annotation on the `Identification`-class?

Comment (cheezi0747) [reply src/main/java/se/sundsvall/comfactfacade/api/model/Signatory.java:26]: You are indeed correct

File: src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java [hunk 2]
```diff
@@ -73,10 +73,10 @@ public interface ComfactClient {
 
 
 	/**
-	 * Get information about a signatory
+	 * Get information about a signatories
```
Comment (mha109) [inline src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java:76]: OCD here `... about signatories`? :)

File: src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java [hunk 3]
```diff
@@ -28,7 +28,7 @@ public interface ComfactClient {
 	 * Create a signing instance
 	 *
 	 * @param createSigningInstance the request to create a signing instance
-	 * return {@link CreateSigningInstanceResponse} containing the signing instance id and URL for each signatory to start their
+	 * return {@link CreateSigningInstanceResponse} containing the signing instance id and URL for each signatories to start their
```
Comment (mha109) [inline src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java:31]: `... each signatory` ?

File: src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java (hunk 2 above)
Comment (cheezi0747) [reply src/main/java/se/sundsvall/comfactfacade/integration/comfact/ComfactClient.java:76]: Something something auto-refactor something. I'll fix

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**